pom.xml

```
@@ -273,6 +273,12 @@
             <type>pom</type>
         </dependency>
 
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-build-step</artifactId>
+            <version>2.14-rc430.69345dedbb03</version>
```

Pending release of https://github.com/jenkinsci/pipeline-build-step-plugin/pull/54
```suggestion
            <version>2.14</version>
```